src/sql/workbench/browser/designer/designerIssuesTabPanelView.ts

```diff
@@ -114,6 +126,15 @@ class TableFilterListRenderer implements IListRenderer<DesignerIssue, DesignerIs
 				break;
 		}
 		templateData.issueIcon.className = `issue-icon ${iconClass}`;
+		if (element.moreInfoLink) {
+			const linkElement = this._instantiationService.createInstance(Link, {
+				label: localize('designer.moreInfoLink', "More information"),
+				href: element.moreInfoLink
+			}, undefined);
+			templateData.issueMoreInfoLink.appendChild(linkElement.el);
+		} else {
+			templateData.issueMoreInfoLink = undefined;
```

Comment:
You shouldn't set it to undefined, next time it is being reused for an issue with more info link, you will get null ref exception when trying to append child to it. The correct thing should be: `DOM.clearNode(templateData.issueMoreInfoLink)`